### PR TITLE
Newsletter

### PR DESCRIPTION
**Pull request by DrCheddar**
Newsletter feature bois

image
image
image
image

### CODE REVIEW

File: components/NotesSidebar.vue [hunk 1]
```diff
@@ -25,9 +25,121 @@
...
+      <!-- <div class="text-xl font-bold w-full vx-row justify-center pb-2">
+        Filters
+      </div> -->
+      
+      <!-- <div
+        class="w-full rounded mb-4"
+        style="background-color: gray; height: 2px;"
+      />
 
+      <h6 class="font-bold mb-3">Rating</h6>
```
Comment (Vansh-Sethi) [inline components/NotesSidebar.vue:37]: Remove all the uncommented code.

File: components/NotesSidebar.vue [hunk 2]
```diff
@@ -93,50 +205,135 @@
...
+
+  subjectGroupClicked(name : SubjectGroup_O)
+  {
+    SubjectGroups[name].forEach((subject : Subject_O)=> this.subjectClicked(subject, false, true));
+  }
+
+
+
   toggleNotesModal(val: boolean) {
+    console.log("banana")
```
Comment (Vansh-Sethi) [inline components/NotesSidebar.vue:268]: Get rid of console.log

File: components/NotesSidebar.vue [hunk 3]
```diff
@@ -93,50 +205,135 @@
...
+    this.hoverStars = this.filterStars = index
+  }
 
   async filterSubjects() {
     const loading = this.$vs.loading()
     notesStore.SetActiveFilter({
       sortSelect : this.sortSelect,
       filterSubjects : this.ActiveSubjectList,
       filterGrades : this.gradeSelect,
+      // allSubjectsSelected : this.allSelected
```
Comment (Vansh-Sethi) [inline components/NotesSidebar.vue:336]: get rid of commented code

File: components/Sidebar.vue [hunk 4]
```diff
@@ -27,20 +30,20 @@
...
-          <div style="">Github</div>
-        </vs-sidebar-item>
+      <vs-sidebar-item href="https://github.com/Quento-Solutions">
+        <template #icon>
+          <i class="sidebar-icon bx bxl-github"></i>
+        </template>
+        <div style="">Github</div>
+      </vs-sidebar-item>
+
+      <vs-sidebar-item href="https://discord.gg/pfyFWus">
```
Comment (Vansh-Sethi) [inline components/Sidebar.vue:40]: Verify that the discord link is a perma link (there's like 5)

File: pages/newsletters.vue [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+.md-container {
+  font-family: 'Quicksand', sans-serif !important;
+  word-wrap: break-word;
+
+  img {
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+  }
+
+  h1,
```
Comment (Vansh-Sethi) [inline pages/newsletters.vue:36]: Put all the selectors on one line

File: pages/newsletters/index.vue [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+                    {{ item.createdAt.toDateString() }}
+                  </div>
+                </div>
+              </div>
+            </div>
+          </nuxt-link>
+        </template>
+      </VxCard>
+    </div>
+    <!-- <vs-button warn @click="toggleNotesModal(true)" class="w-full">
```
Comment (Vansh-Sethi) [inline pages/newsletters/index.vue:68]: commented code, get rid of it

File: pages/newsletters/index.vue [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+    try {
+      await newslettersStore.GetNewsletters()
+    } catch (error) {
+      console.log({ error })
+    }
+    loading.close()
+
+    this.loaded = true
+    return
+    // console.log(this.note)
```
Comment (Vansh-Sethi) [inline pages/newsletters/index.vue:114]: :)

File: pages/newsletters/index.vue (hunk 6 above)
Comment (DrCheddar) [reply pages/newsletters/index.vue:68]: we need that for adding posts - not a better way to do it right now

Comment (DrCheddar) [reply pages/newsletters/index.vue:68]: When I have time I'll write an admin only site as well but for nwo its fine

**Pull request merged**